opentutor_classifier/svm.py

```diff
@@ -0,0 +1,224 @@
+from collections import defaultdict
+from dataclasses import dataclass
+import pandas as pd
+import numpy as np
+from nltk import pos_tag
+from nltk.corpus import stopwords, wordnet as wn
+from nltk.stem import WordNetLemmatizer
+from nltk.tokenize import word_tokenize
+from os import path, makedirs
+import pickle
+from sklearn import model_selection, svm
+from sklearn.metrics import accuracy_score
+from sklearn.preprocessing import LabelEncoder
+from typing import Tuple, Dict
+
+from opentutor_classifier import (
+    AnswerClassifierInput,
+    AnswerClassifierResult,
+    ExpectationClassifierResult,
+    loadData,
+)
+
+
+class SVMExpectationClassifier:
```

Comment:
let's clean up this file (and all python) so that all variables are lowercase snake case. 

The only exception to that rule might be maybe a data class that gets serialized to json

This should be enforced by the lint check, so I will look into making fixing that config too